include/solvers/twostep_time_solver.h

```diff
@@ -78,6 +78,15 @@ class TwostepTimeSolver : public AdaptiveTimeSolver
    */
   virtual void integrate_adjoint_sensitivity(const QoISet & qois, const ParameterVector & parameter_vector, SensitivityData & sensitivities) override;
 
+  /**
+   * A method to compute the adjoint refinement error estimate at the current timestep.
+   * int_{tstep_start}^{tstep_end} R(u^h,z) dt
+   * Fills in an ErrorVector that contains the weighted sum of errors from all the QoIs and can be used to guide AMR.
```

Comment:
If we're using the ARefEE::qoi_set() then "all the QoIs" isn't always correct here.